examples/resources/connectivitymonitor/get_connectivity_monitor.py

```diff
@@ -14,14 +14,29 @@
     ProbeStreamRequest
 )
 from arista.connectivitymonitor.v1 import models
-from cloudvision.Connector.grpc_client import GRPCClient, create_query
+from cloudvision.Connector.grpc_client import create_query
 
+from arista.inventory.v1 import services
+RPC_TIMEOUT = 30  # in seconds
 
 debug = False
 
 parser = argparse.ArgumentParser(add_help=True)
 
 
+def getSerialNumberToHostnameDict(channel):
+    """
+    Return a json dict with the mapping S/N --> hostname of all devices known to CVP.
+    """
+    device_dict = {}
+    stub = services.DeviceServiceStub(channel)
+    # create a stream request
+    get_all_req = services.DeviceStreamRequest()
+     # make the GetAll request and loop over the streamed responses
```

Comment:
./get_connectivity_monitor.py:35:6: E114 indentation is not a multiple of four (comment)